message_parser/src/test/java/cumulus_message_adapter/message_parser/MessageParserTest.java

```diff
@@ -104,6 +104,49 @@ public void testLoadAndUpdateRemoteEvent() {
         }
     }
 
+    @Test
+    public void testLoadAndUpdateRemoteEvent_s6Success() {
+        MessageAdapter messageAdapter = new MessageAdapter();
+        //"Run non-Hanging Test"
+        try {
+            String inputJsonString = AdapterUtilities.loadResourceToString("s6.input.json");
+            // the message is not changed
+            String expectedJsonString = AdapterUtilities.loadResourceToString("basic.input.json");
+
+            Map<String, Object> expectedOutputJson = JsonUtils.toMap(expectedJsonString);
+
+            String taskOutputString = messageAdapter.LoadNestedEvent(inputJsonString, null, null);
+
+            Map<String, Object> taskOutputJson = JsonUtils.toMap(taskOutputString);
+
+        } catch (MessageAdapterException | IOException e) {
+            e.printStackTrace();
+            fail();
+        }
+    }
+
+    @Test
+    public void testLoadAndUpdateRemoteEvent_s6Error() {
+        MessageAdapter messageAdapter = new MessageAdapter();
+        //"Run Hanging Test";
+        try {
+            String inputJsonString = AdapterUtilities.loadResourceToString("s6.input.json");
+            // the message is not changed
+            String expectedJsonString = AdapterUtilities.loadResourceToString("basic.input.json");
```

Comment:
We can actually use this expectation setup in this test:
```suggestion
            String expectedJsonString = AdapterUtilities.loadResourceToString("s6.input.json");
```